ton_sdk/src/tests/test_local_tvm.rs

```diff
@@ -34,6 +34,7 @@ fn test_local_piggy_call() {
         false)
             .expect("Error decoding result");
 
+    println!("Gas fee {}", gas_fee);
```

Comment:
I haven't seen any assertions related to `gas_fee` (print only).

Reply:
Such test is implemented in ton-client-js